### ISSUE TITLE
enforce-consistent-line-wrapping className something changes.

### ISSUE THREAD

**Issue by demonguyj** [schoero/eslint-plugin-better-tailwindcss#299]
### Description

It seems that the way line breaks are handled in className has changed in version 4.0.0 compared to the previous versions (~3.8.0). Is there a way to apply the same line break behavior as before? 
 
- original 
<li
			className="flex-2 overflow-hidden border-b border-solid border-dg-450 pb-9 dark:border-dgdark-450"
		/>

- ~3.8.0
<li
			className={
				`
					flex-2 overflow-hidden border-b border-solid border-dg-450 pb-9
					dark:border-dgdark-450
				`
			}
		/>

- 4.0.0 ~
- <li
			className="
				flex-2 overflow-hidden border-b border-solid border-dg-450 pb-9
				dark:border-dgdark-450
			"
		/>

### Flavor

TSX

### Code Input

```tsx
<li
			className="flex-2 overflow-hidden border-b border-solid border-dg-450 pb-9 dark:border-dgdark-450"
		/>
```

### Expected Behavior

```tsx
<li
			className={
				`
					flex-2 overflow-hidden border-b border-solid border-dg-450 pb-9
					dark:border-dgdark-450
				`
			}
		/>
```

### Actual Behavior

```tsx
<li
			className="
				flex-2 overflow-hidden border-b border-solid border-dg-450 pb-9
				dark:border-dgdark-450
			"
		/>
```

### ESLint Log

```shell
2026-01-16 11:40:03.336 [info] ESLint server stopped.
2026-01-16 11:40:03.834 [info] ESLint server is starting.
2026-01-16 11:40:03.913 [info] ESLint server running in node v22.21.1
2026-01-16 11:40:03.916 [info] ESLint server is running.
```

### ESLint Config

```typescript
{
...
'better-tailwindcss/enforce-consistent-line-wrapping': [
				`error`,
				{
					indent: `tab`
				}
			],
}
```

### Versions

- ESLint: 9.39.2
- Parser: 
- Plugin: 
- Node: 22
